tests/functionaltests/FunctionalTests.cpp

```diff
@@ -54,4 +54,4 @@ MODULE_SETUP(ModuleSetup) {
 
 MODULE_CLEANUP(ModuleCleanup) {
     return SUCCEEDED(FrameworkHelper::DisplayRequest(false));
-}
\ No newline at end of file
+}
```

Comment:
Correct, no more trampolining/connector code required. All nice and self-contained within the mm file.

---
In reply to: [100679868](https://github.com/Microsoft/WinObjC/pull/1947#discussion_r100679868) [](ancestors = 100679868)